src/main/java/uk/gov/hmcts/cmc/claimstore/courtfinder/models/Address.java

```diff
@@ -0,0 +1,17 @@
+package uk.gov.hmcts.cmc.claimstore.courtfinder.models;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+
+import java.util.List;
+
+@JsonIgnoreProperties(ignoreUnknown = true)
+@AllArgsConstructor
+public class Address {
+    @JsonProperty("address_lines")
+    @Getter private List<String> addressLines;
+    @Getter private String postcode;
+    @Getter private String town;
```

Comment:
You can annotate the class itself with `@Getter` to apply it to every non-static field. (Applies to all of these classes.)